<html>
<body>
<p>
This is a Python wrapper for FastTree. Most common options are available in the GUI. For advanced usage, please use the accompanying command-line tool.
</p>
<p>
FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. It can handle alignments with up to a million of sequences in a reasonable amount of time and memory. For large alignments, FastTree is 100-1,000 times faster than PhyML 3.0 or RAxML 7. FastTree is open-source software.
</p>
<p>
For more information, please refer to the official website:<br>
<a href="http://www.microbesonline.org/fasttree/">http://www.microbesonline.org/fasttree/</a>
</p>
</body>
</html>
