<html>
<body>
<p>
This is a Python wrapper and GUI for FastTree. Most common options are available on the sidebar. For more advanced usage, please use the accompanying command-line tool.
</p>
<p>
FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences. It can handle alignments with up to a million of sequences in a reasonable amount of time and memory. For large alignments, FastTree is 100-1,000 times faster than PhyML 3.0 or RAxML 7. FastTree is open-source software.
</p>
<p>
For more information on FastTree, please refer to the <a href="http://www.microbesonline.org/fasttree/">original website</a>.
</p>
<hr style="margin-top: 12px;">
<p>FastTreePy was developed by S. Patmanidis in the framework of the iTaxoTools project.</p>
<p>See the project's repository and the iTaxoTools website here:</p>
<div style="margin-top: 12px; margin-left: 20px; line-height:1.4;">
  <a href="https://github.com/iTaxoTools/FastTreePy/">https://github.com/iTaxoTools/FastTreePy/</a>
  <br>
  <a href="http://itaxotools.org/">http://itaxotools.org/</a>
</div>
<hr style="margin-top: 12px;">
<p>
Vences, M., A. Miralles, S. Brouillet, J. Ducasse, A. Fedosov, V. Kharchev, I. Kostadinov, <br>
S. Kumari, S. Patmanidis, M.D. Scherz, N. Puillandre, S.S. Renner (2021): <br>
iTaxoTools 0.1: Kickstarting a specimen-based software toolkit for taxonomists. - Megataxa 6: 77-92.
</p>


</body>
</html>
